### DISCUSSION TITLE
Arturo availability from common package managers

### DISCUSSION THREAD

**Discussion by demosthenez** [arturo-lang/arturo#1555] [Ideas]
Availability on scoop for windows, for example.

**Comment by RickBarretto** [comment]
@demosthenez , that is good to see more people interested in Arturo!
Thanks to open this discussion. 

First of all, right now, Arturo is available for Homebrew (and it seems that Nimble, we don't know if this works right now, tbh).

That is something that I was talking about with @drkameleon some days ago. (Seriously)!
And we plan to make Arturo available for some others package managers rather than just homebrew soon. (Probably next release).

As taking care of Windows is my responsibility, I was thinking about winget or scoop (or both).
Winget, because it's the default of Microsoft, scoop because I like it's flexibility. 

In fact I barely used them, so, I'm totally open to listen you opinion about this.

**Reply by drkameleon** [reply]
Haha... What a coincidence! Seriously!

But... some motivation to get started is always welcome. Let's go for it! 🚀 

P.S. @demosthenez Welcome to our community! Feel free to come over [our Discord server](https://discord.gg/YdVK2CB) for more (there is quite some movement and it's probably the best place to ask thing and exchange ideas 😉 )
